Sockets/BackEndMonitoringSockets/include/CClientConnectionHandler.h

```diff
@@ -14,14 +14,15 @@ class CClientConnectionHandler
 	bool HandleEvent(const int socket, EEventType type, std::string& message);
 
 private:
-	const int c_max_num_trial = 50;
-	bool HandleRequestEvent(const int socket, EEventType type, 
-		std::string& message);
+	bool HandleRequestEvent(const int socket, EEventType type);
```

Comment:
Still working with raw sockets instead of socket objects?

Reply:
First time when you said me about raw sockets I did not understand. I thought about types of sockets - datagram-sockets, stream-sockets, and raw-sockets. Now I understand and fix it, now I use only class CSocket instead of descriptors